docs/tutorials/working-with-pytrees.md

```diff
@@ -31,7 +31,7 @@ In the context of machine learning (ML), a pytree can contain:
 
 When working with datasets, you can often come across pytrees (such as lists of lists of dicts).
 
-Below is an example of a simple pytree. In JAX, you can use `jax.tree_*`, to extract the flattened leaves from the trees, as demonstrated here:
+Below is an example of a simple pytree. In JAX, you can use {meth}`jax.tree_util.tree_leaves`, to extract the flattened leaves from the trees, as demonstrated here:
```

Comment:
Sorry, just noticed this should be `{func}`, not `{meth}`

Reply:
:+1: Thank you